src/containers/Collection/CollectionList/CollectionList.tsx

```diff
@@ -83,6 +82,10 @@ export const CollectionList = () => {
     : COLLECTION_SEARCH_QUERY_VARIABLES;
   const updateMutation = groups ? UPDATE_COLLECTION_WA_GROUP : UPDATE_COLLECTION_CONTACTS;
 
+  useEffect(() => {
+    setFilter(groups ? WA_GROUPS_COLLECTION : CONTACTS_COLLECTION);
```

Comment:
```suggestion
    setFilter({groupType: groups ? WA_GROUPS_COLLECTION : CONTACTS_COLLECTION});
```

When you add groupType inside the filter prop it creates an new object on every render and calls the filter and count query again.